main.py

```diff
@@ -260,6 +261,11 @@ def parse_configuration(self, configuration: Dict, target_url: str):
             elif key == "sarif" and value is not None:
                 log("Argument 'sarif' is deprecated. Please use --outputFormat='sarif' instead.")
                 sys.exit(1)
+            elif key == "updateAddons":
+                if str.lower(value) == "true":
```

Comment:
```
                self.update_addons = str.lower(value) == "true";
```

Also does python not have a better boolean parse method?

Reply:
Im doing this like that since we already have bool parameters that are treated as strings, the proper way to specify booleans in python should be if `--updateAddons` is specifid is taken as true, if not is false. Remember that at some point we did try it specifyc store_action=true but ending up throwing some errors in some of the QA machines due to incompatibilities with the python version. We can refactor this later since we're already aware it's intended i think it's fine leaving it as it is for now.

Reply:
thats fine i was just curious about pythons parsing functionality, but no big deal.

we definitely should use the ternary instead of the if/else block however, it should be equivalent and if its not then thats a problem

Reply:
done, added the ternary